crisischeckin/CrisisCheckinMobile/CrisisCheckInMobile.sln

```diff
@@ -0,0 +1,101 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
@brunck I'm thinking I was confused and this was a mistake.  I'm using CrisisCheckin.sln to run the mobile solution and CrisisCheckinWeb.sln to run the web solution.  I don't use CrisisCheckInMobile.sln at all.